cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/detector/AnomalyDetector.java

```diff
@@ -295,6 +296,22 @@ private boolean isReadyToFix(Anomaly anomaly) {
       return false;
     }
 
+    private void mayBePersistSelfHealingInfo(Anomaly anomaly, boolean selfHealingStarted) {
+      if (selfHealingStarted) {
+        RESULT_LOG.info(String.format("Started self-healing for [%s] anomaly [%s].",
+                                     getAnomalyType(anomaly).toString(), anomaly.anomalyId()));
```

Comment:
Can we also have anomaly detail here?

Reply:
What other details besides type of anomaly?

Reply:
I mean the anomaly itself,  like [goal violation anomaly](https://github.com/linkedin/cruise-control/blob/2b98cdbe4380159b912dcbcc5a4e3082f52c1843/cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/detector/GoalViolations.java#L104).